mathSection.py

```diff
@@ -7,23 +8,16 @@
 def index():
     strl = ""
     newline = "<br/>"
-    with open("cryptoAPI.txt") as infile:
-        lines = infile.readlines()
-        for x in lines:
-            if x.split()[0] == "Price":
-                price = x.split()[1]
-                strl+="Percent Change Today: "
-                strl+=((percentChange(x.split()[1],x.split()[2])))+newline
-                strl+="Change in Dollars Today: "
-                strl+=((dollarChange(float(x.split()[1]),float(x.split()[2]))))+newline
-            elif x.split()[0] == "Volume":
-                strl+=" Average Volume: "
-                strl+=((averageVolume(x.split()[1:len(x.split())])))+newline
-            elif x.split()[0] == "Shares":
-                shares = x.split()[1]
+    price = CryptoPriceData.CryptoPriceData()
+    price.getData('BTC','USD',0,2)
+    strl+="Percent Change Today: "

```

Comment:
Nice work. I think a method in `CryptoPriceData` that handles string generation would be a good addition. `getMarketCap ` and `percentChange` can be moved over as well.